demo/src/Plugin/DsField/VocabularyTerms.php

```diff
@@ -90,16 +89,16 @@ public function settingsForm($form, FormStateInterface $form_state) {
 
     $names = taxonomy_vocabulary_get_names();
     $vocabularies = Vocabulary::loadMultiple($names); // Should use dependency injection rather.
```

Comment:
Any idea how the container could be inserted here? I've tried, but probably missed sth...